native/acter/src/api/message.rs

```diff
@@ -1252,10 +1246,7 @@ impl RoomMessage {
             event.origin_server_ts.get().into(),
             "m.room.name".to_string(),
         );
-        let body = match event.content.name {
-            Some(name) => format!("changed name to {name}"),
-            None => "changed name".to_string(),
-        };
+        let body = format!("changed name to {}", event.content.name);
```

Comment:
is `event.content.name` not an `Option<String>` anymore?

We changed this so that the format in the timeline isn't those awkward `changed name to Some('new name')` anymore ... If this is still an option please revert this change.

Reply:
`event.content.name` is not an `Option<String>` anymore.